content/release-notes/2023-11-27-storage-and-compute.md

```diff
@@ -0,0 +1,7 @@
+---
+description: Read about the latest fixes and improvements in Neon Serverless Postgres.
```

Comment:
Do we want to use this description to highlight a given feature/fix per release? In aggregate, in an RSS reader, more targetted descriptions might help with scanning the list. Example:
![image](https://github.com/neondatabase/website/assets/5885546/11b8f589-0a8f-4289-ba47-f26ff0650a2b)

Reply:
Good point. Thank you. I have revised the description.